### PR TITLE
Fix footer-map compatibility

### PR DESCRIPTION
**Pull request by Viserius**
1. Footer styles moved to its own stylesheet
2. Map 100% added back in
3. Map and footer work together
4. Fix text of frontpage

### CODE REVIEW

**Comment by Producermark** [comment]
footer is back thank god

File: frontend/src/styles.css [hunk 1]
```diff
@@ -18,4 +18,14 @@ body, html {
 
 .card {
     margin-bottom: 10px;
+}
+
+.bootstrap-grid {
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
+}
```
Comment (killje) [inline frontend/src/styles.css]: The main style.css should not have this specific css, this should be done in the places where it is used

File: frontend/src/app/activity-page/activity-page.component.ts [hunk 2]
```diff
@@ -9,7 +10,9 @@ import { ViewEncapsulation } from '@angular/core';
 })
 export class ActivityPageComponent implements OnInit {
 
-  constructor() { }
+  constructor(private renderer: Renderer2) {
+    this.renderer.addClass(document.body, 'bootstrap-grid');
+   }
```
Comment (killje) [inline frontend/src/app/activity-page/activity-page.component.ts]: This should not have to be added. Maje a container around the html if you need to add a class

**Comment by Producermark** [comment]
FOOTER IS BACK BIATCH

**Pull request merged**